middle_end/flambda2/from_lambda/closure_conversion_aux.ml

```diff
@@ -667,7 +691,12 @@ module Acc = struct
         then { closure_info2 with is_purely_tailrec = false } :: closure_infos2
         else closure_infos
     in
-    closure_info, { t with closure_infos }
+    ( closure_info,
+      { t with
+        closure_infos;
+        code_slot_offsets;
+        slot_offsets = Slot_offsets.empty
```

Comment:
This looks wrong: we lose the slot offsets from before entering the function (or more likely, we store them as part of the function instead of keeping them global)

Reply:
Fixed in 8d63042 : we now keep a stack of slot_offsets